kratos/tests/test_sparse_matrices.py

```diff
@@ -109,6 +109,18 @@ def test_matrix_assembly(self):
             for i in range(len(validation_index1)):
                 self.assertEqual(B_scipy.indptr[i], validation_index1[i])
 
+        #test transpose and TransposeSpMV
+        x.SetValue(1.0)
+        y.SetValue(0.0)
+        A.TransposeSpMV(x,y) 
+
+        At = A.Transpose()
+        y2 =  KratosMultiphysics.SystemVector(y.size())
+        y2.SetValue(0.0)  
+        At.SpMV(x,y2)
+        for i in range(y.Size()):
+            self.assertEqual(y[i], y2[i], 1e-14)
```

Comment:
there is also `assertVectorAlmostEqual` :)

https://github.com/KratosMultiphysics/Kratos/blob/6d6d5b943ded5c524753fa6807633312665a3e50/kratos/python_scripts/KratosUnittest.py#L76